code/analysis/python/python/read_cff_files.py

```diff
@@ -19,14 +28,17 @@ def read_cff_files(datadir):
                 # invalid file list
                 try:
                     cff_file = yaml.safe_load(f)
-                    cff_data.append(cff_file)
+                    if sanity_check(cff_file):
+                        _cff_data.append(cff_file)
+                    else:
+                        _invalid_file.append(file)
                 except:
-                    invalid_file.append(file)
-    return cff_data, invalid_file
+                    _invalid_file.append(file)
```

Comment:
Is it worth separating files which do not have the correct information, from ones where an exception is called?

Reply:
Yes, agree. I'll adapt.

Reply:
Should be fixed in 70ebf99674d24acf4b123712005199edb7cece5d.